src/umem/mem/mmap.rs

```diff
@@ -67,7 +67,11 @@ mod inner {
             let err = unsafe { libc::munmap(self.addr.as_ptr(), self.len) };
 
             if err != 0 {
-                error!("`munmap()` failed with error code {}", err);
+                error!(
```

Comment:
Good spot! Do we need `err` in the message though, since I guess it'd always be -1? Also no need for the `std` prefix since `std::io` is imported up top.
```suggestion
                error!(
                    "`munmap()` failed with error: {}",
                    io::Error::last_os_error()
                );
```